Cargo.toml

```diff
@@ -18,4 +19,4 @@ debug = false
 debug-assertions = false
 
 [patch.crates-io]
-cosmos-sdk-proto = { git = "https://github.com/forbole/cosmos-rust", branch = "main"}
+cosmos-sdk-proto = { git = "https://github.com/shravanshetty1/cosmos-rust", branch = "wasm2"}
```

Comment:
Can we use the upstream version instead of the your fork?

Reply:
The upstream version is not wasm compatible

Reply:
Working on getting it merged -
https://github.com/cosmos/cosmos-rust/pull/97
https://github.com/cosmos/cosmos-rust/pull/90
https://github.com/cosmos/cosmos-rust/pull/88